examples/src/bin/embassy_serial.rs

```diff
@@ -83,7 +84,25 @@ async fn main(spawner: Spawner) {
     let timg0 = TimerGroup::new_async(peripherals.TIMG0, &clocks);
     embassy::init(&clocks, timg0);
 
-    let mut uart0 = Uart::new_async(peripherals.UART0, &clocks);
+    let io = Io::new(peripherals.GPIO, peripherals.IO_MUX);
+
+    // Default pins for Uart/Serial communication
+    #[cfg(feature = "esp32")]
```

Comment:
couldn't we just use the type alias from uart.rs?

Reply:
ah sorry ... we need the pin not just the type here ... I should probably drink a second coffee